package/kernel/linux/modules/video.mk

```diff
@@ -294,6 +294,20 @@ endef
 
 $(eval $(call KernelPackage,drm-amdgpu))
 
+define KernelPackage/drm-i915
+  SUBMENU:=$(VIDEO_MENU)
+  TITLE:=i915 DRM support
+  DEPENDS:=@TARGET_x86 @DISPLAY_SUPPORT +kmod-drm-kms-helper
+  KCONFIG:=CONFIG_DRM_I915=m
```

Comment:
Why is this module needed, the driver is already compiled into the x86 images:
```
hauke@hauke-t480:~/openwrt/openwrt$ git grep CONFIG_DRM_I915=y
target/linux/x86/64/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/64/config-5.4:CONFIG_DRM_I915=y
target/linux/x86/generic/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/generic/config-5.4:CONFIG_DRM_I915=y
target/linux/x86/legacy/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/legacy/config-5.4:CONFIG_DRM_I915=y
```

Reply:
replaced i915 part with vboxvideo module